mgmtsystem/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Management System -  Base
+=========================
+
+This module is the basis of any management system applications:
+     * audit reports,
+     * nonconformities,
+     * immediate actions,
+     * preventive actions,
+     * corrective actions,
+     * improvement opportunities.
+
+Installation
+============
+
+Makes the Management System base Application available so that other Management System can be insalled
+
+Configuration
+=============
+
+No configuration required
+
+Usage
+=====
```

Comment:
What do I expect in the Usage section? If I try it on runbot, a guide to where should I go and what should I do to check the module features.